### PR TITLE
Configure updated

### PR DESCRIPTION
**Pull request by mridulnagpal**

### CODE REVIEW

File: plinth/frontpage.py [hunk 1]
```diff
@@ -27,7 +27,8 @@ def get_shortcuts():
     return sorted(shortcuts.values(), key=lambda item: item['label'])
 
 
-def add_shortcut(id, label, url, icon, details=None, login_required=False):
+def add_shortcut(id, label, url, icon, details=None, details_front="Hello",
```
Comment (jvalleroy) [inline plinth/frontpage.py]: details_front should have default value of None.

File: plinth/modules/minetest/__init__.py [hunk 2]
```diff
@@ -51,6 +51,10 @@
           'is needed.'), box_name=_(cfg.box_name)),
 ]
 
+description_frontpage = [
+    _('<a href=\"apps/minetest\">Configure</a>'),
```
Comment (jvalleroy) [inline plinth/modules/minetest/__init__.py]: Notice how each description_frontpage has some things in common:
* Starts with '<a href=\"apps/'.
* Ends with '\">Configure</a>'.
* Although it's a list, each one only has one element.
So I suggest that the HTML tag and "Configure" text should be moved to the index template. The only variable here is the URL.

File: plinth/templates/index.html [hunk 3]
```diff
@@ -70,6 +70,15 @@ <h3 class="panel-title">{{ details_label }}</h3>
           {{ paragraph|safe }}
         </div>
       {% endfor %}
+
+      {% if user.is_authenticated %}
+        {% for paragraph in configure_url %}
```
Comment (jvalleroy) [inline plinth/templates/index.html]: If we change configure_url to not be a list (as I suggested above), then we won't need the for loop here.

**Comment by mridulnagpal** [comment]
@jvalleroy Done with the changes please review

**Comment by jvalleroy** [comment]
@mridulnagpal I don't see the changes, did you push to the same branch?

**Comment by mridulnagpal** [comment]
@jvalleroy Resent a PR #657

**Pull request closed without merging**